lnplus/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_LNPLUS_IP
```

Comment:
As static container IPs are no longer a requirement, we can instead put `lnplus_web_1` here